### ISSUE TITLE
Bug in insult feature

### ISSUE THREAD

**Issue by hitenSharm** [hitenSharm/Discord-Meme-and-Music-Bot#11]
Sometimes a text related bug happens.
![image](https://user-images.githubusercontent.com/56029311/135799334-7aa4b7d0-b382-4fba-84a7-e07d8c5c8dcb.png)
Maybe if we change this to Discord MessageEmbed this will not happen.
Whatever the case, change it to  [MessageEmbed](https://discordjs.guide/popular-topics/embeds.html) so that it looks better.

**Comment by rish78** [comment]
I would like to work on this issue. But I am rather new to discord bots, so I'll need some guidance, especially on how to set up the bot on my local machine.

**Comment by hitenSharm** [comment]
It's not that difficult. Take a look at the readme for the setup and take a look at articles online regarding Discord bot tokens.
Make sure to have your node at the latest(not the latest stable) version(V16 as of now).  If you wish to contact me, then take a look at my website!

**Comment by rish78** [comment]
Yup, thanks! I was able to set it up locally. I think I would like to try my hands out on the issue. Could you assign it to me please?
